src/libc/src/test-comp.c

```diff
@@ -2,7 +2,7 @@
 
 _Bool __VERIFIER_nondet_bool(void) {
   _Bool var = owi_i32();
-  owi_assume(or_(var == 0, var == 1));
+  owi_assume(var == 0 || var == 1);
```

Comment:
@filipeom, I thought we were not using `or_` and `and_` anymore but actually we are using it there, do you think this is going to have a negative impact in some way to change it like this ? Maybe, we could also expose a `owi_bool()` function instead, what do you think ?

Reply:
It will depend on how the compiler decides to compile this. But most likely it will try to lazy evaluate the condition, possibly causing unnecessary branches. Creating  a `owi_bool` is cleaner. But, using a bitwise-or or chaining two `owi_assume` would most-likely also work to reduce branching also.

Reply:
@Laplace-Demon, do you want to try to add a `owi_bool` function ?